sources/sala/graphemes.tsv

```diff
@@ -17,10 +17,10 @@ BIPA	GRAPHEME	ID	SYMBOLS
 <NA>	   	T11NL	
 <NA>	   	T12NL	
 <NA>	   	T13NL	
-<NA>	ƈ	C189NL	
-<NA>	ƙ	C191NL	
-<NA>	ƥ	C177NL	
-<NA>	ƭ	C183NL	
+ʄ̊	ƈ	C189NL	

```

Comment:
```
$ clts table ʄ̊
INFO    cldf-clts/clts at /home/mattis/data/datasets/cldf/clts
# consonant
```
| GRAPHEME   | PHONATION   | PLACE   | MANNER    | ALIAS   | EXTRA            | NOTE   |
|:-----------|:------------|:--------|:----------|:--------|:-----------------|:-------|
| ʄ̥ / ʄ̊      | voiced      | palatal | implosive | +       | voicing:devoiced |        |


This means: the norm that we expect is to put the little ball under the symbol. If you want to put it above, you need to add the table (shown above) to the consonants.tsv.